### DISCUSSION TITLE
Empty brackets in "Distribution"-key's value. What should be there?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [hardinfo2/hardinfo2#65] [Q&A] [answered]
> ![Screenshot_20240817_234449](https://github.com/user-attachments/assets/92b99aa7-445d-46bd-9bcd-7a8e3f6245f9)

**Comment by hwspeedy** [comment, marked as answer]
It's the codename, as you can see the source code.
Fedora has codename as ""

Improved by this patch: https://github.com/hardinfo2/hardinfo2/commit/392af2f36bd5fd1ea6a2365ef8f0f173878d5344

**Reply by roke-julian-lockhart** [reply]
> https://github.com/hardinfo2/hardinfo2/discussions/65#discussioncomment-10380412

@hwspeedy, seems that Fedora doesn't use codenames anymore: https://www.reddit.com/r/Fedora/s/VW54nEHZm9. Thanks for the change.
